fedimint-client/src/client.rs

```
@@ -1406,9 +1419,22 @@ impl Client {
                     let invite_code = self.invite_code(req.peer).await;
                     yield serde_json::to_value(invite_code)?;
                 }
+                "get_operation" => {
+                    let req: GetOperationIdRequest = serde_json::from_value(params)?;
+                    let operation = self.operation_log().get_operation(req.operation_id).await;
+                    yield serde_json::to_value(operation)?;
+                }
                 "list_operations" => {
-                    // TODO: support pagination
-                    let operations = self.operation_log().paginate_operations_rev(usize::MAX, None).await;
+                    let req: ListOperationsParams = serde_json::from_value(params)
+                        .unwrap_or_default();
```

don't hide errors with unwrap_or_default use ?